### DISCUSSION TITLE
Desmiolada

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#326] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/desmiolada/

**Comment by maryon-telvanni** [comment]
Nossa! Fizeram uma *webcomic* sobre mim! :grin:

**Comment by angellisy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/desmiolada/#comment_1142114419>):

> Nossa! Fizeram uma *webcomic* sobre mim! 😁

:rofl:

**Comment by brunoaugusto** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/desmiolada/#comment_1142114419>):

> Nossa! Fizeram uma *webcomic* sobre mim! 😁

![null](<https://i.postimg.cc/CxVf3NHL/i-know-that-feel-butt-squeeze.jpg>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/desmiolada/#comment_1142262021>):

> [![null](<https://camo.githubusercontent.com/b37099f9b2e4e3c44f127544a33d2ad5ffc1c80d64799a029da3c8091106da64/68747470733a2f2f692e706f7374696d672e63632f43785666334e484c2f692d6b6e6f772d746861742d6665656c2d627574742d73717565657a652e6a7067>)](<https://camo.githubusercontent.com/b37099f9b2e4e3c44f127544a33d2ad5ffc1c80d64799a029da3c8091106da64/68747470733a2f2f692e706f7374696d672e63632f43785666334e484c2f692d6b6e6f772d746861742d6665656c2d627574742d73717565657a652e6a7067>)

![null](<https://i.imgur.com/mn7pS8z.png>)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/desmiolada/#comment_1142262021>):

> [![null](<https://camo.githubusercontent.com/b37099f9b2e4e3c44f127544a33d2ad5ffc1c80d64799a029da3c8091106da64/68747470733a2f2f692e706f7374696d672e63632f43785666334e484c2f692d6b6e6f772d746861742d6665656c2d627574742d73717565657a652e6a7067>)](<https://camo.githubusercontent.com/b37099f9b2e4e3c44f127544a33d2ad5ffc1c80d64799a029da3c8091106da64/68747470733a2f2f692e706f7374696d672e63632f43785666334e484c2f692d6b6e6f772d746861742d6665656c2d627574742d73717565657a652e6a7067>)

:rofl:
